src/rocksdb_store.rs

```diff
@@ -36,27 +34,14 @@ pub enum SetError {
 }
 
 impl RocksDBStore {
-    pub(crate) fn new(config: &StoreConfig) -> Self {
+    pub(crate) fn new(constructor_bundle: Location) -> Self {
```

Comment:
rename to location

Reply:
This has already been done. I pushed it....

Reply:
```suggestion
    pub(crate) fn new(location: Location) -> Self {
```
And rest of the method